src/Components/MovieList.js

```diff
@@ -3,10 +3,20 @@ import PropTypes from 'prop-types';
 import MovieCard from './MovieCard';
 import '../Sass/MovieList.scss';
 import { formatRating } from '../utilities';
+import Dropdown from './Dropdown';
 
-const MovieList = ({ movies, filteredMovies, icon }) => {
+const MovieList = ({ movies, filteredMovies, icon, stateChange }) => {
 
   const moviesToDisplay = filteredMovies.length ? filteredMovies : movies;
```

Comment:
This is the reason the full list is displayed when nothing comes back from a search. I think we'll need to pass error from state as a prop into this so we can check if filteredMovies.length AND if the error state has a message "No movies found" if both of those are true, then show nothing, otherwise show the complete list.